metaflow/plugins/argo/argo_workflows.py

```diff
@@ -187,6 +232,12 @@ def _sanitize(name):
     def delete(name):
         client = ArgoClient(namespace=KUBERNETES_NAMESPACE)
 
+        workflow_template = argo_client.get_workflow_template(name)
+        sensor_name = name.replace(".", "-")
```

Comment:
sensor name is also recorded in the template, but we cannot use that here because of the legacy case where it was not recorded yet, correct?

Reply:
Good point, I will get the actual sensor name from the annotations and then use this as the fallback.